start_db.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+set -e
+
+POSTGRES_OPTIONS="-c config_file=/etc/postgresql/postgresql.conf"
+
+# Externally mounted volumes are owned by root, make sure the data directory
+# is owned by postgres user inside the container
+echo "\n===== Modifying ownership/permissions of data directories ====="
+chown -R postgres "/var/lib/postgresql"
+
+# Check if this is the first time running
+if [ ! -s "$PGDATA/PG_VERSION" ]; then
+
+    echo "\n============= Initializing Postgres Data Directory ============"
+    gosu postgres initdb
+
+    echo "\n=============== Starting Postgres in Background ==============="
+    gosu postgres postgres $POSTGRES_OPTIONS &
+    POSTGRES_PID="$!"
+    sleep 5  # wait for postgres to start up before using psql
```

Comment:
That looks really promising, I was mostly going off of the existing postgres instructions, but `pg_ctl` looks more appropriate